source/includes/gcp/_k8_daemonsets.md

```diff
@@ -79,7 +79,7 @@ Note that the list is not complete, since it is refering to the [kubernetes api
 ```shell
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/v1/services/k8s/an_environment/statefulsets/test-aerospike/auth"
+   "https://cloudmc_endpoint/v1/services/k8s/an_environment/daemon sets/test-aerospike/auth"
```

Comment:
Should we add an underscore in `daemon sets` (like `daemon_sets`) as URLs can't contain spaces?

Reply:
Sure thing! I've updated it